source/faq.txt

```diff
@@ -99,29 +99,29 @@ What is a discriminator?
 A discriminator is a property that identifies a specific document
 schema. You can use it for inheritance and storing multiple types of
 documents within the same collection or parent document (if you embed
-sub-documents). 
+sub-documents).
 
 For example, if you have an ``Event`` class that you extend in Java (e.g.
 ``MachineEvent`` or ``NetworkEvent``), using a discriminator identifies
 which class the ``PojoCodecProvider`` should use to serialize/deserialize the
-document. 
+document.
 
 For more information, see our :ref:`POJO Customization guide
-<pojo-discriminators>`. 
+<pojo-discriminators>`.
 
 Can I control serialization of ``LocalDate``?
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 Yes, the 3.7 Java driver adds native support for ``JSR-310 Instant``,
-``LocalDate`` & ``LocalDateTime``. 
+``LocalDate`` & ``LocalDateTime``.
 
 Can I serialize a ``java.util.Date`` as a string in format **yyyy-mm-dd**?
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-Yes, you can build your own codec for this class and add it to the registry. 
+Yes, you can build your own codec for this class and add it to the registry.
 
-Add the codec to the top your registry before the ``DefaultCodecRegister``
-and before the ``PojoCodecProvider``: 
+Add the codec to the top your registry before the default codec registry
```

Comment:
I agree with the first change.
From the second part of the comment, I'm not sure whether you saw the sample code below that demonstrates it since it might be below the fold. I was originally just correcting an issue I noticed as I was going through some of the existing content. I spotted a couple other issues which I'll correct in the next commit, but just want to make sure I'm not missing anything from your comment.